src/index.html

```diff
@@ -1,13 +1,44 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./index.css">
+  </head>
+  <body>
+    <div data-qa="card" class="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="imac"
+        class="card_logo"
+      >
+      <h2 class="card_name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card_code">Product code: 195434</p>
+
+      <div class="card__rating">
+        <div class="stars">
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__review">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <span class="card__price-title">Price</span>
+        <span class="card__price-amount">$2,199</span>
+      </div>
+      <a
+        href="https://www.apple.com/ua/imac-24/"
+        data-qa="hover"
+        class="card__buy-button">Buy
+      </a>
```

Comment:
Pay attention to formating HTML!
```suggestion
      <a
          href="https://www.apple.com/ua/imac-24/"
          data-qa="hover"
          class="card__buy-button"
        >
          Buy
      </a>
```